docs/rules/no-access-state-in-setstate.md

```diff
@@ -0,0 +1,39 @@
+# Prevent using this.state within a this.setState (no-access-state-in-setstate)
+
+This rule should prevent usage of this.state inside setState calls.
+Such usage of this.state might result in errors when two state calls is
```

Comment:
Yes, I can probably do that for all `this.state` and `setState` mentions in the docs. Thanks, I will fix that first thing to the morning tomorrow.